packages/assistify/package.js

```diff
@@ -17,6 +17,7 @@ Package.onUse(function(api) {
 
 	//Server
 	api.addFiles('config.js', 'server');
+	//TODO add jquery?
```

Comment:
the question was about jquery which is used by the bundled smarti script. It is included/packaged there which is maybe also not a good solution. requirejs would be the best solution?